pages/linux/pyrit.md

```diff
@@ -0,0 +1,36 @@
+# pyrit
+
+> WPA/WPA2 cracking tool using computational power.
+> More information: <https://github.com/JPaulMora/Pyrit>.
+
+- Display system cracking speed:
+
+`pyrit benchmark`
+
+- List available cores:
+
+`pyrit list_cores`
+
+- Set ESSID:
+
+`pyrit -e {{ESSID}} create_essid`
+
+- Read and analyze packet capture file:
+
+`pyrit -r {{path/to/file}} analyze`
+
+- Read and import passwords to the database:
+
+`pyrit -i {{path/to/file}} import_passwords`
```

Comment:
```suggestion
- Read and [i]mport passwords to the current database:

`pyrit -i {{path/to/file}} {{import_unique_passwords|unique_passwords|import_passwords}}`
```

Almost the same issue as above.
Optional fix: as database placeholder is not found, it's assumed that this example referred to the current database.